protocols/raft/src/main/java/io/atomix/protocols/raft/session/RaftSession.java

```diff
@@ -205,7 +207,7 @@ public long getRequestSequence() {
    * @return the next request sequence number
    */
   public long nextRequestSequence() {
-    return this.requestSequence + 1;
+    return this.requestSequence.get() + 1;
```

Comment:
get isn't thread safe according to the javadocs, https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/atomic/LongAccumulator.html#get--  then again the previous implementation wasn't thread safe either.

Reply:
Hm, I originally figured that it's fine if it doesn't return the latest value, and more importantly just that it cannot set a lower value. I think in most cases it should be fine, but that probably breaks the principle of least astonishment. Easy fix here would be to convert to an `AtomicLong`.

Reply:
Or use a volatile long with AtomicLongFieldUpdater?

Reply:
I'm not really familiar with using `AtomicLongFieldUpdater`; what's the difference with `AtomicLong`? Isn't an `AtomicLong` just a volatile long with some CAS methods?